wondrous-app/components/Common/TaskSubmission/submissionItem.tsx

```diff
@@ -542,61 +546,99 @@ export function SubmissionItem({
-            />
-          </SubmissionReviewButtons>
-          <ReopenTaskSubmission submission={submission} setCommentType={setCommentType} onClick={reopenTask} />
-          <SubmissionBountyPaymentButton
-            fetchedTask={fetchedTask}
-            submission={submission}
-            fetchedTaskSubmissions={fetchedTaskSubmissions}
-            handleClose={handleClose}
-            getTaskSubmissionsForTask={getTaskSubmissionsForTask}
-          />
+            <SubmissionEditButton isCreator={isCreator} approvedAt={submission.approvedAt} onClick={handleEdit} />
+          </Grid>
+          {!hideTaskSubmissionButtons ? (
+            <Grid
+              item
+              sm={8}
+              xs={12}
+              paddingY={{ sm: '0px', xs: '10px' }}
+              borderTop={{ sm: 'none', xs: '1px solid #343434' }}
+            >
+              <ResubmitTaskSubmissionButton
+                submission={submission}
+                setShowComments={setShowComments}
+                setShowCommentBox={setShowCommentBox}
+                setCommentType={setCommentType}
+                commentType={commentType}
+                isCreator={isCreator}
+                onClick={resubmitTaskSubmission}
+              />
+            </Grid>
+          ) : null}
+          {!hideTaskStatusButtons ? (
+            <Grid
+              item
+              sm={8}
+              xs={12}
+              borderTop={{ sm: 'none', xs: '1px solid #343434' }}
+              paddingY={{ sm: '0px', xs: '10px' }}
+            >
+              <SubmissionReviewButtons canReview={canReview} fetchedTaskStatus={fetchedTask?.status}>
+                <SubmissionRejectButton
+                  submission={submission}
+                  rejectTaskSubmission={rejectTaskSubmission}
+                  commentType={commentType}
+                />
+                <SubmissionRequestChangeButton
+                  submission={submission}
+                  setShowComments={setShowComments}
+                  setShowCommentBox={setShowCommentBox}
+                  setCommentType={setCommentType}
+                  commentType={commentType}
+                />
+                <SubmissionApproveTaskButton
+                  submission={submission}
+                  fetchedTaskType={fetchedTask?.type}
+                  setCommentType={setCommentType}
+                  setShowComments={setShowComments}
+                  setShowCommentBox={setShowCommentBox}
+                  approveSubmission={approveSubmission}
+                  commentType={commentType}
+                />
+                <SubmissionApproveBountyButton
+                  submission={submission}
+                  fetchedTaskType={fetchedTask?.type}
+                  onClick={approveBountySubmission}
+                  commentType={commentType}
+                />
+              </SubmissionReviewButtons>
+            </Grid>
+          ) : null}
+          {isSubmissionStatusUpdated ? (
```

Comment:
we shouldnt use "? : "here, easier to just do &&

Reply:
I can update it to use `&&` if you'd prefer `&&` but it's not a better choice especially when rendering components

Reply:
See attached. That's why I always avoid using `&&`.
![image](https://user-images.githubusercontent.com/35099553/203417072-e3f8af7e-d54c-4a28-920b-ed5d9db3b600.png)